inference/core/workflows/core_steps/loader.py

```diff
@@ -1,135 +1,149 @@
-    DetectionsFilterBlock,
+from inference.core.workflows.core_steps.models.third_party.barcode_detection.version_1 import (
+    BarcodeDetectorBlockV1,
 )
-from inference.core.workflows.core_steps.transformations.detections_transformation import (
-    DetectionsTransformationBlock,
+from inference.core.workflows.core_steps.models.third_party.qr_code_detection.version_1 import (
+    QRCodeDetectorBlockV1,
 )
-from inference.core.workflows.core_steps.transformations.dynamic_crop import (
-    DynamicCropBlock,
+from inference.core.workflows.core_steps.sinks.roboflow.custom_metadata.version_1 import (
+    RoboflowCustomMetadataBlockV1,
 )
-from inference.core.workflows.core_steps.transformations.dynamic_zones import (
-    DynamicZonesBlock,
+from inference.core.workflows.core_steps.sinks.roboflow.dataset_upload.version_1 import (
+    RoboflowDatasetUploadBlockV1,
 )
-from inference.core.workflows.core_steps.transformations.perspective_correction import (
-    PerspectiveCorrectionBlock,
+from inference.core.workflows.core_steps.transformations.absolute_static_crop.version_1 import (
+    AbsoluteStaticCropBlockV1,
 )
-from inference.core.workflows.core_steps.transformations.relative_static_crop import (
-    RelativeStaticCropBlock,
+from inference.core.workflows.core_steps.transformations.detection_offset.version_1 import (
+    DetectionOffsetBlockV1,
 )
-from inference.core.workflows.core_steps.visualizations.background_color import (
-    BackgroundColorVisualizationBlock,
+from inference.core.workflows.core_steps.transformations.detections_filter.version_1 import (
+    DetectionsFilterBlockV1,
+)
+from inference.core.workflows.core_steps.transformations.detections_transformation.version_1 import (
+    DetectionsTransformationBlockV1,
+)
+from inference.core.workflows.core_steps.transformations.dynamic_crop.version_1 import (
+    DynamicCropBlockV1,
+)
+from inference.core.workflows.core_steps.transformations.dynamic_zones.version_1 import (
+    DynamicZonesBlockV1,
+)
+from inference.core.workflows.core_steps.transformations.perspective_correction.version_1 import (
+    PerspectiveCorrectionBlockV1,
+)
+from inference.core.workflows.core_steps.transformations.relative_static_crop.version_1 import (
+    RelativeStaticCropBlockV1,
+)
+from inference.core.workflows.core_steps.visualizations.background_color.version_1 import (
+    BackgroundColorVisualizationBlockV1,
 )
 
 # Visualizers
-from inference.core.workflows.core_steps.visualizations.blur import (
-    BlurVisualizationBlock,
+from inference.core.workflows.core_steps.visualizations.blur.version_1 import (
+    BlurVisualizationBlockV1,
+)
+from inference.core.workflows.core_steps.visualizations.bounding_box.version_1 import (
+    BoundingBoxVisualizationBlockV1,
 )
-from inference.core.workflows.core_steps.visualizations.bounding_box import (
-    BoundingBoxVisualizationBlock,
+from inference.core.workflows.core_steps.visualizations.circle.version_1 import (
```

Comment:
I'm wondering if it would be more elegant to keep the same convention we have in versioning the engine in file system (i.e. `v1` instead of `version_1`)

Reply:
👍